internal/sql/repository/chartConfig/ChartRepository.go

```diff
@@ -18,11 +18,48 @@
 package chartConfig
 
 import (
+	"fmt"
 	"github.com/devtron-labs/devtron/internal/sql/models"
 	"github.com/devtron-labs/devtron/internal/sql/repository"
 	"github.com/go-pg/pg"
 )
 
+type BatchOperationExample struct {
```

Comment:
table should folllow apiVersion, kind, subresource structure

Reply:
will this be part of this file? why inside of ChartRepository?

Reply:
made new repository for BulkUpdate, this table is used for getting see example content...thus the task, script and readme structure